web/source/osk/oskView.ts

```diff
@@ -17,6 +17,12 @@ namespace com.keyman.osk {
     'nomove'?: boolean
   };
 
+  export enum ActivationMode {
+    static      = "static",  // For use by documentation keyboards, eventually.
+    manual      = "manual",
+    conditional = "conditional"
```

Comment:
I'd prefer to use our old name of 'automatic' which I think reflects better that Keyman is maintaining the visibility of the keyboard -- 'conditional' implies that the developer will be able to set conditions on when it displays.